pages/infrastructure_providers-validators/running_full_node.md

```diff
@@ -1,73 +1,60 @@
-# Running a Full Node
-
-### Save your Chain ID in `dydxprotocold` config
-
-Save the [chain-id](../infrastructure_providers-network/network_constants.mdx#chain-id). This will make it so you do not have to manually pass in the chain-id flag for every CLI command.
-
-```bash
-dydxprotocold config chain-id $CHAIN_ID
+# Run a Full Node
+Running a full node allows your system to participate in a dYdX chain network. The recommended settings and best practices on this page help ensure your node stays healthy and up to date with the network.
```

Comment:
1. nit: "participate in a network" makes it sound like it's part of block production. is there a better wording to clarify this more?
2. do we need to clarify why someone would want to run a full node in more details? for example, why does someone want to "participate" in the network? examples could be that they want to submit orders, read orderbook data, etc
3. do we clarify full node (aka non-validating node) vs validator node somewhere? not necessary for this PR but would be nice to think about how to make such clarification and why MMs should run the former (and not the latter)

Reply:
1. Fair point, I'll address this with 2:

2. I'll replace "participates in a.. network" with essentially a preview of next steps, saying something like  "read data directly from the chain, build a local orderbook to integrate with programmatically, and place, confirm, and cancel orders."

3. We do not, or at least not in a high enough spot to catch people looking for that. I think that the first two pages in this section, hardware and config requirements, could all be merged into one page that explains the difference between validators and full nodes, lists prereqs that apply to both, then links readers to "Set up a Full Node" or the equivalent validator page (right now there's only one on running, not setting up). Linear ticket: https://linear.app/dydx/issue/CT-1051/add-top-level-info-to-full-nodes-and-validators-explaining-the